vendor/github.com/ethereum/go-ethereum/whisper/whisperv5/whisper.go

```diff
@@ -79,6 +79,7 @@ type Whisper struct {
 
 	mailServer         MailServer // MailServer interface
 	envelopeTracer     EnvelopeTracer // Service collecting envelopes metadata
+	maxPeers           int32
```

Comment:
Any particular reason using `int32` instead of `int`?

Reply:
i wanted to use smaller int, maybe int8

Reply:
Not sure it has any implication except for data type casting when using `len` and comparing values, which we do later.